README.md

```diff
@@ -21,11 +21,7 @@ While developed by Googlers, Mobly is not an official Google product.
 
 ## Compatibility
 
-Mobly is compatible with both *python 3.4+* and *python 2.7*.
-
-NOTE: As part of the bigger [communitiy initiative](https://python3statement.org/),
-we are planning to drop Python 2.7 support in 2020. Mobly 1.10 is likely the last
-major release to support Python 2.7.
+Mobly is compatible with both *python 3.6+*.
```

Comment:
no more "both"

Let's say, `Mobly requires Python 3.6 or newer`

Reply:
Thank you for this comment.